routes/events.js

```diff
@@ -7,16 +7,25 @@ module.exports = function(app) {
     // Route for creating events
     app.post('/events/create', function(req, res, next) {
 
-        eventServices.createEvent(req.body, function(err, event) {
+    var auth_token = req.get('Authorization');
+    jwt.verify(auth_token, app.get('superSecret'), function(error, userId) {
+        User.findById(userId, function(error, user) {
+            eventServices.createEvent(req.body, user.tier, function(err, event) {
```

Comment:
you'll want to check the `err` object here as well (and elsewhere if there are more istances; i won't make any more comments :P)